index.html

```diff
@@ -0,0 +1,422 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css"
+     integrity="sha512-1ycn6IcaQQ40/MKBW2W4Rhis/DbILU74C1vSrLJxCq57o941Ym01SwNsOMqvEBFlcgUa6xLiPY/NS5R+E6ztJQ==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link href="https://fonts.googleapis.com/css2?family=Bungee+Spice&family=Inter:wght@100;200;300;400;500;600;700;800;900&family=Lato:wght@300;400;700;900&family=Lobster+Two:wght@400;700&family=Poppins:wght@300;400;500;600;700;800&family=Satisfy&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Online Conference</title>
+</head>
+<body>
+    <main id="main-content">
+    <header>
+        <header class="nav-social">
+            <nav class="nav-social">
+                <ul class="list-icon">
+                    <li class="soc-icon">
+                        <a rel="noopener" target="_blank" href="https://web.facebook.com/microverseinc/?_rdc=1&_rdr"><i class="fab fa-facebook-f"></i></a>
+                      </li>
+                      <li class="soic-icon">
+                        <a rel="noopener" target="_blank" href="https://twitter.com/microverseinc"><i class="fab fa-twitter"></i></a>
+                      </li>
+                      <li>English</li>
+                      <li>My Page</li>
+                      <li>Logout</li>
+                </ul>
+            </nav>
+        </header>
+        
+        <header class="header-desk">
+            
+            <nav class="heads-all">
+                <ul>
+                    <li class="logo-img">
+                        <a href="./index.html">
+                            <img src="./img/CC_Global_Summit_2015 logo.png" alt="collision logo">
+                        </a>
```

Comment:
- [ ] `required`
- [ ] Please make the first navbar black like the template
- [ ] also the logo should be in line with where you start the paragraph `face to face with gorillas`
- [ ] and I suggest you increase the fonts like the original template
- [ ] please screenshots below

`your template`
![image](https://user-images.githubusercontent.com/66230306/227656878-f695a422-1348-47ac-b429-95df8d63e02e.png)

`original template`
![image](https://user-images.githubusercontent.com/66230306/227657009-6e25ff75-ac11-40f2-bfb7-1098f3978263.png)